### PR TITLE
common-main.c: fflush stdout buffer when exit

### PR DESCRIPTION
**Pull request by chengzhuo5**

### CODE REVIEW

File: common-main.c [hunk 1]
```diff
@@ -86,5 +86,10 @@ int common_exit(const char *file, int line, int code)
 	check_bug_if_BUG();
 	trace2_cmd_exit_fl(file, line, code);
 
+	/**
+	 * Some software (such as security software) may change buffer type of stdout.
+	 * It will no output without fflush manually.
+	 */
+	fflush(stdout);
```
Comment (dscho) [inline common-main.c:93]: Good! Could you move this to the beginning of the function, though?

**Review by dscho — APPROVED** [review]
Great finding, and obviously-good fix!

You may want to follow the advice in https://github.blog/2022-06-30-write-better-commits-build-better-projects/ to align the commit message's style with Git's conventions.

File: common-main.c (hunk 1 above)
Comment (chengzhuo5) [reply common-main.c:93]: Thanks for your advice. I will change my commit soon.

**Comment by dscho** [comment]
Thank you @chengzhuo5 for going the extra mile to fix the issue!

**Comment by Saschaym** [comment]
chengzhuo5:main

**Pull request merged**